eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
If you reviewer can agree my opinion, I will add `README.md` to both projects to explain their responsibility.